discovery_engine_flutter/example/pubspec.yaml

```diff
@@ -6,7 +6,8 @@ description: Demonstrates how to use the xayn_discovery_engine_flutter plugin.
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 
 environment:
-  sdk: ">=2.17.3 <3.0.0"
+  sdk: '>=2.17.3 <3.0.0'
+  flutter: '>=3.0.2'
```

Comment:
Let's use the relaxed versions like the app to avoid problem with their CI
flutter: '>=3.0.0'
sdk: '>=2.17.0 <3.0.0'